### DISCUSSION TITLE
Hi Johan I see that you made some good progress keep up

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages-server#22] [General]
Hi Johan,
This message is just to encourage you and make you cheer up. 
I know you can do it.
Keep going step by step issue after issue pull request after pull request.
To give us this awesome powerful new project from the joomlatools team.

**Comment by johanjanssens** [comment]
Thanks @alexandreelise much appreciated.

**Reply by johanjanssens** [reply]
@alexandreelise Joomlatools Server is now ready for Joomla 3 and 4.

**Reply by alexandreelise** [reply]
4.1.3?

**Reply by johanjanssens** [reply]
Any Joomla version, it will install the latest Joomla 4 release for you automatically.

**Reply by alexandreelise** [reply]
Even Joomla 1.0? Just kidding.

**Reply by alexandreelise** [reply]
I'm pretty sure you have a multi billion dollar client running Joomla 1.0

**Reply by johanjanssens** [reply]
To clarify, any Joomla version that support PHP8.

**Reply by alexandreelise** [reply]
Enjoy your day Master J.
